dask/dataframe/tseries/resample.py

```diff
@@ -32,8 +32,12 @@ def _resample_series(
     )
     return out.reindex(
         pd.date_range(
-            start, end, freq=rule, closed=reindex_closed, name=out.index.name
-        ),
+            start.tz_localize(None),
+            end.tz_localize(None),
+            freq=rule,
+            closed=reindex_closed,
+            name=out.index.name,
+        ).tz_localize(start.tz, nonexistent="shift_forward"),
```

Comment:
@TomAugspurger This "nonexistent" parameter

Reply:
I'm OK with matching the behavior of the pandas version in use by the user here. Given that, I think that

1. For pandas 0.24 and newer, we drop the tz for start and end and call tz_localize with `nonexistent="shift_forward")
2. For pandas 0.23.x and older, we retain the behavior on master.

`dask.dataframe._compat` defines some helpers for pandas versions.

Then the test you've added will only be run for pandas 0.24 and newer.

Reply:
Okey... Now fixed